packages/browser/src/ClientAuthentication.ts

```diff
@@ -112,17 +113,16 @@ export default class ClientAuthentication {
   // if the expected information cannot be found.
   // Note that the ID token is not stored, which means the session information
   // cannot be validated at this point.
+  // FIXME: not actually part of the public interface:
   validateCurrentSession = async (
     currentSessionId: string
   ): Promise<(ISessionInfo & ISessionInternalInfo) | null> => {
     const sessionInfo = await this.sessionInfoManager.get(currentSessionId);
-    if (
-      sessionInfo === undefined ||
-      sessionInfo.clientAppId === undefined ||
-      sessionInfo.issuer === undefined
-    ) {
+
+    if (sessionInfo === undefined) {
```

Comment:
sessionInfo is undefined if the issuer is undefined, and clientAppId is no longer used here (see splitting sessionInfoManager and ClientRegistrar)